### ISSUE TITLE
M1 Mac, Instant crashing on startup

### ISSUE THREAD

**Issue by brandonmrgich** [AllTheMods/ATM-9#330]
After updating to the newest version of ATM9, when attempting to generate a new world, the game would crash. After a few attempts at this, it started crashing just on the startup of the pack. Though this was a Java issue, tried removing and uninstalling Zulu17 and verifying Java installation, though crashes continued. Not sure what's doing this. 

NOTE: I updated and ran this mod pack side by side on my m1 Mac, and a Windows machine. It did load on the Windows machine, however, there was some lag and rendering issues.

Here is the log that was generated on the M1, if you require any other logs or information, please let me know!

[hs_err_pid19410.log](https://github.com/AllTheMods/ATM-9/files/12513920/hs_err_pid19410.log)

**Comment by screret** [comment]
I believe M1 macs kinda just have shit openGL/java/anything support
You might be out of luck, though someone else might know a solution

**Comment by pavelbeard** [comment]
how it's going with this problem? I have the same...

**Comment by tht** [comment]
Have you tried this hint? It solved the issue for me: https://github.com/AllTheMods/ATM-9/issues/407#issuecomment-1858712053

tldr: Remove the Spark Plugin and start again.

**Comment by tht** [comment]
Probably a duplicate of #407

**Comment by brandonmrgich** [comment]
> Have you tried this hint? It solved the issue for me: [#407 (comment)](https://github.com/AllTheMods/ATM-9/issues/407#issuecomment-1858712053)
> 
> tldr: Remove the Spark Plugin and start again.

This is it, I managed to have it running good with this too!

**Comment by Zajozor** [comment]
Can confirm removing Spark fixes issue.

Macbook Pro 2020 (M1), with zulu's java 17.

#1031 and #407 
are probably the same problem.
